packages/core/src/combo-box/useComboBox.tsx

```diff
@@ -70,7 +70,7 @@ export function useComboBox<Item>(props: UseComboBoxProps<Item>) {
     setSelectedState(newSelected);
     const newValue = getOptionsMatching((option) =>
       newSelected.includes(option.value)
-    ).map((option) => listControl.valueToString(option.value));
+    ).map((option) => listControl.valueToString(option.value.value));
```

Comment:
Don't know the intent for the change, `.value.value` means one of the variable naming is not great?

Reply:
It's a breaking change to change it now